ui/tests/settings_user.spec.ts

```diff
@@ -1,5 +1,14 @@
 import {Application, SpectronClient} from 'spectron';
-import {closeAddYourSettingsPopup, createAccount, GLOBAL_TIMEOUT, initialiseSpectron, METHOD_TIMEOUT, navigateTo} from './common';
+import {
+    captureOnFailure,
```

Comment:
Why are all these changes in these .ts files happening? Does your editor have some different linting options than mine?

Reply:
Usually most of the linters I know off will start breaking the import if you are over the line length.
So for me it doesn't seem wrong. What happens if you run the formatter here?